masterlist.yaml

```diff
@@ -18727,69 +18851,35 @@ plugins:
-      - <<: *patch3rdParty
-        subs:
-          - 'Cloaks of Skyrim'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Cloaks.esp") and not active("JKs Temple of Talos - Cloaks of Skyrim patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Enhanced Lights and FX'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("EnhancedLightsandFX.esp") and not active("JKs Temple of Talos - ELFX Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Interesting NPCs'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("3DNPC.esp") and not active("JKs Temple of Talos - 3DNPC patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Oblivion Artifact Pack'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("WZOblivionArtifacts.esp") and not active("JKs Temple of Talos - Oblivion Artifacts patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Palaces and Castles Enhanced'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Palaces Castles Enhanced.esp") and not active("JKs Temple of Talos - PCE patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Snazzy Furniture and Clutter Overhaul (SFCO)'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Snazzy Furniture and Clutter Overhaul.esp") and not active("JKs Temple of Talos - SFCO patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Solitude and Temple Frescoes 2019'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("SolitudeTempleFrescoes.esp") and not active("JKs Temple of Talos - Solitude and Temple Frescoes Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'The Elder Scrolls Online Imports'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("ESOimports.esp") and not active("JKs Temple of Talos - ESO Imports Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Unofficial Skyrim Special Edition Patch'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Unofficial Skyrim Special Edition Patch.esp") and not active("JKs Temple of Talos - USSEP patch.esp")'
+      - <<: *compatPatch_JKIPC
+        subs: [ '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)' ]
```

Comment:
This can be refactored with the following:

```yaml
    msg: [ *compatPatch_JKIPC ]
```

Sorry, GH's suggestions aren't supported when there would be line deletions in them, so you'll have to do this manually.

Reply:
I think this shows as resolved on my end so I'm clicking Resolve thanks for all your help with these

Reply:
It's still there, you'll have to fix it in your clone & push the commit.

Reply:
I don't know what I'm doing wrong.  My version shows it as finished even here on Github

[https://i.imgur.com/86QyuJz.png](url)

https://github.com/cecell/skyrimse/blob/v0.17/masterlist.yaml